src/extra/draw/draw_text.ts

```diff
@@ -0,0 +1,182 @@
+import type { CanvasRenderingContext2D as SkiaCanvasRenderingContext2D } from 'skia-canvas';
+
+import type { Image, ImageDataArray } from '../../Image.ts';
+import type { Point } from '../../geometry/index.ts';
+import { getCanvasContext } from '../../utils/cross_platform.ts';
+import { getOutputImage } from '../../utils/getOutputImage.ts';
+import { validateValues } from '../../utils/validators/validators.ts';
+
+export interface DrawTextLabel {
+  /**
+   * Text of the label.
+   */
+  content: number | string;
+  /**
+   * Position to draw the label at.
+   */
+  position: Point;
+  /**
+   *  Size and type of font.
+   */
+  font?: string;
+  /**
+   *  Font color. Should be in rgba8 format.
+   */
+  fontColor?: number[];
```

Comment:
In my previous comment I meant that those 2 options can be both in the label and in the general options.

These options take precedence and general options are a default fallback. The behavior should be documented in the jsdoc.